conductr_cli/sandbox_main.py

```diff
@@ -97,6 +104,16 @@ def build_parser():
                             default=False,
                             dest='no_wait',
                             action='store_true')
+    run_parser.add_argument('--interface',
+                            default=DEFAULT_SANDBOX_INTERFACE,
+                            help='The network interface which will be used by ConductR Sandbox to bind to')
+    run_parser.add_argument('--addr-range',
+                            type=addr_range,
+                            default=DEFAULT_SANDBOX_ADDR_RANGE,
+                            help='Range of address which will be used by ConductR Sandbox to bind to')
```

Comment:
Make mention that we use the [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing).